skimage/feature/_texture.pyx

```diff
@@ -92,7 +85,7 @@ cdef inline int _bit_rotate_right(int value, int length) nogil:
     return (value >> 1) | ((value & 1) << (length - 1))
 
 
-def _local_binary_pattern(double[:, ::1] image,
+def _local_binary_pattern(cnp.float64_t[:, ::1] image,
```

Comment:
Why change this?

Reply:
i don't know. gardening. Raymond Hettinger taught me not to do it. I'll revert.